lib/check_result_utils.go

```diff
@@ -12,43 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package checker
+package lib
```

Comment:
`utils` are recommended in go

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
There was another comment from @inferno-chromium mentioning generic names are to be avoided in Go. So going back to checker/ for now.